_source/logzio_collections/_prometheus-sources/dotnet-custom.md

```diff
@@ -561,5 +562,154 @@ using (var stream = new MemoryStream())
 </div>
 <!-- tab:end -->
 
+
+<!-- tab:start -->
+<div id="troubleshooting">
+
+This section contains some guidelines for handling errors that you may encounter when trying to collect .NET metrics.
+
+## Problem: No metrics received
+
+No metrics are observed in your Logz.io account.
+
+### Possible cause - Incorrect token and/or listener URL
+
+Your Logz.io token and/or listener URL may be incorrect.
+#### Suggested remedy
+
+1. Navigate to  **[Manage tokens](https://app.logz.io/#/dashboard/settings/manage-tokens/shared) > [Data shipping tokens - Metrics](https://app.logz.io/#/dashboard/settings/manage-tokens/data-shipping?product=metrics)** and verify your account's metrics token and listener URL.
+
+2. Check in the integration code whether the token and listener URL are specified correctly.
+
+### Possible cause - Shipper connectivity failure
+
+Your Logz.io listener may not be connected to the network.
+
+#### Suggested remedy
+
+Verify connectivity of your Logz.io listener as follows.
+
+* For Linux and Mac servers, use `telnet`:
+
+  ```shell
+  telnet listener.logz.io <<PORT>>
+  ```
+
+
+* For Windows servers running Windows 8/Server 2012 and later, use the following command in PowerShell:
+
+```shell
+Test-NetConnection listener.logz.io -Port <<PORT>>
+```
+
+Replace `<<PORT>>` with the appropriate port nummber. For HTTPS communication use port 8053. For HTTP communication use port 8052.
+
```

Comment:
Can you please indent the shell and the line after to be inside the bullet point